scripts/travis/build_and_test.sh

```diff
@@ -37,7 +37,13 @@ if [[ "$DO_BUILD" == "yes" ]] ; then
       or_die ctest -T test --output-on-failure -V
     fi
     if [[ "${DO_MEMCHECK}" == "yes" ]] ; then
-      or_die ctest -T memcheck
+      regex="^Memory Leak - [1-9]+"
+      while read -r line; do
+          if [[ "$line" =~ $regex ]]; then
+            echo "Found leaks: $line"
```

Comment:
Not really sure this is sufficient.  [https://travis-ci.com/LLNL/Umpire/jobs/207604652](https://travis-ci.com/LLNL/Umpire/jobs/207604652) reports a leak, but I cannot tell which leak it is?  Could we say what the line number is so we can go back in the build log and look for context?

Reply:
The problem is that you can't get context without dumping all the valgrind logs. I could have it do that on error?

Reply:
I think that would be good.  I am not sure what _Memory Leak - 8_ means.

Reply:
Since it is reporting one now, can we try it and see what it looks like?

Reply:
It means there are 8 leaks 🙄

Reply:
Well that was obvious.  Would love to see what they are :-)

Reply:
Much better!